website/docs/reference/node-selection/syntax.md

```diff
@@ -91,8 +91,9 @@ dbt can leverage artifacts from a prior invocation as long as their file path is
 - [The `state:` selector](/reference/node-selection/methods#the-state-method), whereby dbt can identify resources that are new or modified
 by comparing code in the current project against the state manifest.
 - [Deferring](/reference/node-selection/defer) to another environment, whereby dbt can identify upstream, unselected resources that don't exist in your current environment and instead "defer" their references to the environment provided by the state manifest.
+- The [`dbt clone` command](website/docs/reference/commands/clone.md), whereby dbt can clone nodes based on their location in the manifest provided to the `--state` flag.
 
-Together, these two features enable ["slim CI"](/guides/legacy/best-practices#run-only-modified-models-to-test-changes-slim-ci). We expect to add more features in future releases that can leverage artifacts passed to the `--state` flag.
+Together, the `state:` selector and deferral enable ["slim CI"](/guides/legacy/best-practices#run-only-modified-models-to-test-changes-slim-ci). We expect to add more features in future releases that can leverage artifacts passed to the `--state` flag.
```

Comment:
I wrote this back in Oct 2020 (https://github.com/dbt-labs/docs.getdbt.com/pull/391/commits/019600dad61a561e13d74ba5a1618ee1918c21ba):

> We expect to add more features in future releases that can leverage artifacts passed to the `--state` flag.

look at us, we're doing it :)